elearning-frontend/src/components/admin/Users.js

```diff
@@ -133,49 +133,57 @@ const Users = () => {
-                                                    avatar: (response.data.user.profile.avatar !== null && response.data.user.profile.avatar != "" ? response.data.user.profile.avatar : null),
-                                                })
-                                
-                                                if (response.data.user.profile.avatar !== null && response.data.user.profile.avatar != "") {
-                                                    setHasAvatar(true)
-                                                }
-                                            }).catch(error => {
-                                                Toastify("error", error)
-                                            })
-                                        }}>EDIT</button>
-                                        <button onClick={(e) => {
-                                            e.preventDefault()
-
-                                            if (window.confirm('Are you sure you want to delete this user?')) {
                                                 apiClient({
-                                                    method: "delete",
+                                                    method: "get",
                                                     url: "/api/admin/user/" + user.id,
-                                                    headers: {
-                                                        'Content-Type': 'multipart/form-data'
-                                                    }
                                                 }).then(response => {
-                                                    Toastify("success", "Succesfully deleted user")
+                                                    ClearFields()
+
+                                                    setHasSelectedUser(true)
+                                                    setSelectedUser(response.data.user.id)
+                                                    setUser({
+                                                        ...user,
+                                                        ...response.data.user.profile,
+                                                        profilepic: (response.data.user.profile.avatar !== null && response.data.user.profile.avatar != "" ? 'http://127.0.0.1:8000/uploads/avatar/' + response.data.user.profile.avatar : defaultPicture),
+                                                        avatar: (response.data.user.profile.avatar !== null && response.data.user.profile.avatar != "" ? response.data.user.profile.avatar : null),
+                                                    })
+                                    
+                                                    if (response.data.user.profile.avatar !== null && response.data.user.profile.avatar != "") {
+                                                        setHasAvatar(true)
+                                                    }
```

Comment:
it is not good to look at. Just set all the response to the state like this.
`setuser(response.data.user)`
Note that this is a repeat comment.

Then you can do the conditional rendering avatar mismo sa avatar component. like this

`{user.profile.avatar ? <Img src ..../> : <AvatarDummyImage />}`